packages/desktop-client/src/components/payees/PayeeTableRow.tsx

```diff
@@ -122,7 +135,58 @@ export const PayeeTableRow = memo(
         }}
         data-focus-key={payee.id}
         onMouseEnter={() => onHover && onHover(payee.id)}
+        onContextMenu={e => {
+          e.preventDefault();
+          setMenuOpen(true);
+          const rect = e.currentTarget.getBoundingClientRect();
+          setCrossOffset(e.clientX - rect.left);
+          setOffset(e.clientY - rect.bottom);
+        }}
       >
+        <Popover
+          triggerRef={triggerRef}
+          placement="bottom start"
+          isOpen={menuOpen}
+          onOpenChange={() => setMenuOpen(false)}
+          crossOffset={crossOffset}
+          offset={offset}
+          style={{ width: 200, margin: 1 }}
+          isNonModal
+        >
+          <Menu
+            items={[
+              payee.transfer_acct == null && {
+                name: 'delete',
```

Comment:
When checking with `null` I prefer using `==` as it also matches `undefined`. Which is in this project plausable because TS checking is not as strict.